docs/home/howto/ISTIO.md

```diff
@@ -1,115 +0,0 @@
-## Istio
```

Comment:
Is there a reference to this file, HOWTO.md for example?

Reply:
I removed all of the references to Istio in our docs (apart from design docs which will be cleared in another task).